data/scripts.js

```diff
@@ -1507,6 +1507,11 @@ exports.BattleScripts = {
 			moves[3] = 'hypervoice';
 			hasMove['hypervoice'] = true;
 		}
+		if (template.id === 'meloettapirouette' && !hasMove['relicsong']) {
+			delete hasMove[this.getMove(moves[3]).id];
+			moves[3] = 'relicsong';
+			hasMove['relicsong'] = true;
+		}
```

Comment:
Bleh, I'm future proofing in case Meloetta gets some other good Physical move.

Did you know that the randbats script (singles and doubles) gives the Flame Orb to Pokemon with Marvel Scale and Psycho Shift? No Pokemon has that ability/move combination, yet there it is.